docker-compose.yaml

```diff
@@ -8,6 +8,7 @@ services:
     environment:
       - POSTGRES_USER=${DB_USERNAME}
       - POSTGRES_PASSWORD=${DB_PASSWORD}
+      - POSTGRES_DB=${DB_NAME}
```

Comment:
é necessário essa mudança ? Não creio que usamos essa env var

Reply:
Essa variável de ambiente é usada pela imagem do postgres pra definir um nome diferente para o banco de dados padrão criado quando a imagem é iniciada pela primeira vez. Se não for especificado, o valor de `POSTGRES_USER` ("user") será usado.

Como temos uma variável de ambiente pro nome do banco no .env, e é usada no momento da conexão, interessante garantir que o banco default criado tenha o mesmo nome.